Documentation/specs/runtime-configuration-file.md

```diff
@@ -117,8 +122,8 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 
 This section is copied verbatim from an identical section in the input `project.json` file (with the exception of the `target` parameter which is generated by the compilation process). The `runtimeConfig` section specifies parameters to be provided to the runtime during initialization. Known parameters include:
 
-* `gcServer` - Boolean indicating if the server GC should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/ms229357.aspx) setting)
-* `gcConcurrent` - Boolean indicating if background garbage collection should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/yhwwzef8.aspx) setting).
+* `System.GC.Server` - Boolean indicating if the server GC should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/ms229357.aspx) setting)
```

Comment:
Instead of describing the options here, it might be better to have a link to https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/clr-configuration-knobs.md, which will be updated to describe each exposed option. A link in the default config file may also be useful.